manifest.json

```diff
@@ -7,13 +7,13 @@
     },
     "url": "https://github.com/YunoHost-Apps/rocketchat_ynh",
     "license": "GPL-3.0",
-    "version": "0.64.0",
+    "version": "0.64.1",
     "maintainer": {
         "name": "selamanse",
         "email": "selamanse@scheinfrei.info",
         "url": "https://github.com/selamanse"
     },
-    "multi_instance": false,
+    "multi_instance": true,
     "services": [
```

Comment:
Leaving this as default might have implications (see rocketchat MongoDB advise on multi instance setup). 

Also: does it even make sense to install rocketchat as multiple services on a ynh instance? I left this option to decide if this service might be installed multiple times on ynh (one master installing the dB and others in slave mode connecting to an existing db)

Let’s discuss..

Reply:
My idea is to install it on multiple domains. 
On mattermost I use 2 teams, one for family and one for friends. Since rocket chat doesn't have multiple teams, I would install 2 instances on different domains.

Do you have a link about the multiple instance setup ?

Reply:
Sure here is the link:

https://rocket.chat/docs/installation/manual-installation/multiple-instances-to-improve-performance/

I think what you want to achieve is not possible with adding instances (which is only for bigger teams). You will still have one Domain  (unlike Mattermost or Slack)

Reply:
It is not really the same purpose.
I think loadblancing would be harder because we need to automatically detect how many instance we got to modify the nginx settings.
It can be a future evolution.
But I still think multi-instance can be usefull

Reply:
Yes I agree, I have opened issue #48 to address this separately (because the setup does not use this option yet, it was just a prepartion from my side).

If you revert this from true to false I'd like to merge this PR.